backends/config/config_test.go

```diff
@@ -149,6 +149,8 @@ func TestNewBaseBackend(t *testing.T) {
 			desc:     "Redis",
 			inConfig: config.Backend{Type: config.BackendRedis},
 			expectedLogEntries: []logEntry{
+				{msg: "Error creating Redis backend: At least one Host[s] is required.", lvl: logrus.FatalLevel},
+				{msg: "Creating Redis backend", lvl: logrus.InfoLevel},
 				{msg: "Error creating Redis backend: ", lvl: logrus.FatalLevel},
```

Comment:
In practice, `logrus.FatalLevel` stops execution. Under this scenario, we would only see the first entry in the logs:
```
148         {
149             desc:     "Redis",
150             inConfig: config.Backend{Type: config.BackendRedis},
151             expectedLogEntries: []logEntry{
152                 {msg: "Error creating Redis backend: At least one Host[s] is required.", lvl: logrus.FatalLevel},
153 -               {msg: "Creating Redis backend", lvl: logrus.InfoLevel},
154 -               {msg: "Error creating Redis backend: ", lvl: logrus.FatalLevel},
155             },
156         },
backends/config/config_test.go
```

Reply:
I thought the same but without those three items the test fail.

Reply:
It's because the test function disables the `logrus` exit functions, so we can have as many Fatal calls without interrupting the test:
```
func TestNewBaseBackend(t *testing.T) {
	// logrus entries will be recorded to this `hook` object so we can compare and assert them
	hook := logrusTest.NewGlobal()
	defer func() { logrus.StandardLogger().ExitFunc = nil }()
	logrus.StandardLogger().ExitFunc = func(int) {}
```

Reply:
Good catch, thanks @sebmil-daily